switchmap/server/db/attributes.py

```
@@ -206,6 +207,25 @@ def resolve_organization(obj, _):
     return obj.organization.decode() if bool(obj.organization) else ""
 
 
+def resolve_device_by_hostname(obj, info, hostname=None):
```

Ruff (0.13.3)

Unused function argument: `obj`

(ARG001)